v22.2/use-cloud-storage-for-bulk-operations.md

```diff
@@ -138,6 +138,115 @@ You [can associate an EC2 instance with an IAM role](https://docs.aws.amazon.com
 aws ec2 associate-iam-instance-profile --iam-instance-profile Name={example profile} --region={us-east-2} --instance-id {instance example}
 ~~~
 
+### Assume role authentication
+
+{{site.data.alerts.callout_info}}
+CockroachDB supports assume role authentication on clusters running v22.2. Authenticating to cloud storage with `ASSUME_ROLE` on clusters running versions v22.1 and earlier, or mixed versions, is not supported and will result in failed bulk operations.
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v22.2" %} To limit the control over Amazon S3 buckets, you can create IAM roles for users to assume. IAM roles do not have an association to a particular user. The role contains permissions that define the operations a user (or [Principal](https://docs.aws.amazon.com/IAM/latest/UserGuide/intro-structure.html#intro-structure-principal)) can complete. An IAM user can then assume a role to undertake a CockroachDB backup, restore, import, etc. As a result, the IAM user only has access to the assigned role, rather than having unlimited access to an S3 bucket. 
+
+{{site.data.alerts.callout_success}}
+Role assumption applies the principle of least privilege rather than directly providing privilege to a user. Creating IAM roles to manage access to AWS resources is Amazon's recommended approach compared to giving access straight to IAM users.
+{{site.data.alerts.end}}
+
+For example, if you want to provide a user access to complete a bulk operation to an Amazon S3 bucket, take the following steps:
```

Comment:
nit: This is an assume-specific example, so we need a tiny mention that this example does indeed have that as its specific target scope:

> For example, to configure a user to use an assumed IAM role to complete a bulk operation to an Amazon S3 bucket, take the following steps: